src/main/java/frc/team3388/robot/subsystems/IntakeSystem.java

```diff
@@ -0,0 +1,35 @@
+package frc.team3388.robot.subsystems;
+
+import com.flash3388.flashlib.io.devices.DoubleSolenoid;
+import com.flash3388.flashlib.io.devices.SpeedController;
+import com.flash3388.flashlib.scheduling.Subsystem;
+
+public class IntakeSystem extends Subsystem {
+
+    private SpeedController motor;
+
+    private DoubleSolenoid pistons;
+
+
+
+    public IntakeSystem(SpeedController motor, DoubleSolenoid pistons) {
+        this.motor = motor;
+        this.pistons = pistons;
+    }
+
+    public void collect(double speed) {
```

Comment:
You should not receive a parameter. This system is meant to only move at a specific speed. What you had earlier was the intention.

Reply:
yeah thats related to the other thing